### DISCUSSION TITLE
Why do I get multiple icons of "Notes Review Queue"?

### DISCUSSION THREAD

**Discussion by holliebutler** [st3v3nmw/obsidian-spaced-repetition#1135] [Q&A] [answered]
I'm a new user trying to figure out the documentation to get this plugin to work (it'd be so helpful if documentation didn't assume that people already had so much technical knowledge), and I _think_ I have a basic deck functioning, except that every time I restart Obsidian, the plugin adds an additional "Notes Review Queue" icon to the top right hand corner of the app. 

Why is it doing that and how do I get it to stop?  

<img width="279" alt="Screenshot 2024-10-16 at 1 31 12 PM" src="https://github.com/user-attachments/assets/964f7031-85a8-48ae-838f-50f5b392b480">

**Comment by chrsdk** [comment]
Most recent version of Obsidian broke a few plugins. Temporary workaround is to rollback to the previous version of Obsidian.

**Comment by st3v3nmw** [comment, marked as answer]
Hi, I've released v1.13.2 which should fix the issue (a new icon being added every time Obsidian is opened).
For the existing icons, please remove them manually i.e. Right click on the icon then click Close.

**Reply by st3v3nmw** [reply]
> it'd be so helpful if documentation didn't assume that people already had so much technical knowledge

I'm planning to update the documentation towards the end of the year to make it easier for new users.
